e2e-tests/finalizer/conf/some-name.yml

```diff
@@ -0,0 +1,85 @@
+apiVersion: psmdb.percona.com/v1
+kind: PerconaServerMongoDB
+metadata:
+  name: some-name
+  finalizers:
+    - delete-psmdb-pods-in-order
```

Comment:
We are not testing this finalizer, but currently I don't have some smart way to check it.

Reply:
Me too. But I add it as we have it as default configuration.